schemas/epos-msl-0/uischema.json

```diff
@@ -37,15 +37,19 @@
     }
   },
   "GeoLocation": {
-    "ui:description": "Geographical location refers to a specific physical point on Earth. This is more precise than 'area' or 'place' often defined by a set of latitude and longitude coordinates.",
+    "ui:description": "Sample location: the location of the outcrop/quarry/borehole/other from which the sample(s) used in your study were taken originally. Please do not add a location of a shop, repository or other facility that temporarily stores samples. Model location: include only if a model you publish refers to a specific location on Earth.",
     "items": {
       "geoLocationBox": {
         "ui:field": "geo"
       }
     }
   },
-  "Keyword": {
-    "ui:description": "Free text field for adding (searchable) keywords to your data package"
+  "HierarchicalKeyword": {
+    "ui:description": "Adding (searchable) keywords to your data package.",
+    "ui:field": "hierarchical_keyword_selector",
+    "ui:data": "https://raw.githubusercontent.com/UtrechtUniversity/msl_vocabularies/refs/heads/main/vocabularies/combined/editor/1.3/editor_1-3.json",
```

Comment:
What should be our url for EPOS MSL hierarchy? This link or the epos-msl-keywords.json file we added in this branch?

Reply:
This link, the one in this repo was just for testing purposes.